packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -390,6 +471,13 @@ export interface MenuProps {
 	 * The set of Containers to offer as selection options.
 	 */
 	containers?: ContainerKey[];
+
+	/**
+	 * Callback function when a container should be removed from the list.
+	 *
+	 * @defaultValue `undefined` - No remove functionality is available when not provided.
+	 */
+	onRemoveContainer?: (containerKey: ContainerKey) => void;
```

Comment:
Nit: since we no longer anticipate _giant_ lists of containers (as we did when you were prototyping the data object based solution), we can probably do without this change. If we are going to keep it, I would prefer _always_ allowing users to remove containers. It seems a bit odd to me to _only_ allow users to remove containers that are closed.

Reply:
Does this actually dispose of the container? I assumed this was just removing the container's devtools instance from our list? I'm not suggesting we give UX for disposing of the container in the Containers list, just that if we offer the ability to deregister a closed container from here, we should let them deregister any container.

Reply:
Ah you're right. Nevermind, it doesn't so we should just provide the remove option for every registered instances.